ui/client/containers/SearchFilter.js

```diff
@@ -0,0 +1,23 @@
+import filterIcon from "../assets/img/search.svg"
```

Comment:
I think this is wrong space (/client/containers/) for SearchFilter component..

Reply:
Done